Core/Lib/ExtendedController/ComercialContactController.php

```diff
@@ -42,6 +42,24 @@ abstract class ComercialContactController extends EditController
      * @param string $viewName
      */
     abstract protected function setCustomWidgetValues($viewName);
+    
+    /**
+     * Check that the subaccount length is correct.
+     * 
+     * @param  integer $lenghtsubcuenta
+     */
+    protected function checkLengthSubaccount($lenghtsubcuenta)
+    {   
+        $codempresa = $this->request->cookies->get('fsCompany');
```

Comment:
No uses la cookie para obtener la empresa. Ya tienes la empresa en $this->empresa

Reply:
Hecho